dex/networks/eth/contracts/ETHSwapV0.sol

```diff
@@ -60,17 +60,6 @@ contract ETHSwap {
         require(block.timestamp >= refundBlockTimestamp);
         _;
     }
-
-    // isRedeemable checks that a swap can be redeemed. The requirements are
-    // the participant is msg.sender, the state is Filled, and the passed secret
-    // hashes to secretHash.
-    modifier isRedeemable(bytes32 secretHash, bytes32 secret) {
```

Comment:
The swaps map is public, so I think it is possible to determine redeemability, but an exported public function is still fine to add imo. Maybe better because it will use solidity's logic, which may be different somehow than what we expect.

Reply:
I agree with @JoeGruffins, we could just look into the swaps map. Adding the the isRedeemable function would cost about an extra $25 when deploying the contract if gas price is 100 gwei, so I guess it's not too much for the added convenience.